non-critical-infra/modules/mailserver/default.nix

```diff
@@ -9,8 +9,29 @@
 
     fqdn = config.networking.fqdn;
 
-    # TODO: change to `nixos.org` when ready
-    domains = [ "mail-test.nixos.org" ];
+    domains = [ "nixos.org" ];
+  };
+
+  sops.secrets."nixos.org.mail.key" = {
```

Comment:
error: builder for '/nix/store/kjy9nba2i4xyyc9zgd7i80bac6wprh4p-manifest.json.drv' failed with exit code 1;
       last 1 log lines:
       > /nix/store/v2v880irn2ww28p0cwjvxcrhw51daj6a-sops-install-secrets-0.0.1/bin/sops-install-secrets: manifest is not valid: secret nixos.org.mail.key in /nix/store/3b7xm21cgqjhnqfras21b8gywc2iqvvi-nixos.org.mail.key.umbriel is not valid: the key 'nixos.org.mail.key' cannot be found
       For full logs, run:

Reply:
Yep, sorry! Was building locally when I pushed. Should have waited.

Fixed now.